dataminer-overview/Pricing/Pricing_Perpetual_Use_Licensing.md

```diff
@@ -88,7 +98,7 @@ The [SLA Manager App](https://community.dataminer.services/business-intelligence
 
 ### Custom apps
 
-In addition to the standard core DataMiner licenses, which allow anybody to design, build, and deploy comprehensive multi-vendor end-to-end network management and orchestration solutions, you can also opt to license custom, purpose-built DataMiner applications from Skyline Communications.
+In addition to the standard core DataMiner licenses, which allow anybody to design, build, and deploy comprehensive multi-vendor end-to-end network management and orchestration solutions, you can also opt to license custom, purpose-built DataMiner applications from Skyline Communications, example [DataMiner PTP app] (https://skyline.be/precision-time-protocol).
```

Comment:
For this link, I'd prefer to link to [https://docs.dataminer.services/user-guide/Standard_Apps/PTP/SolPTP.html](https://docs.dataminer.services/user-guide/Standard_Apps/PTP/SolPTP.html), as internal links within dataminer.docs are more future-proof. If you link to an outside page and that page is removed, we will not get a warning that the link no longer works.

```suggestion
In addition to the standard core DataMiner licenses, which allow anybody to design, build, and deploy comprehensive multi-vendor end-to-end network management and orchestration solutions, you can also opt to license custom, purpose-built DataMiner applications from Skyline Communications, such as the [DataMiner PTP app](xref:SolPTP).
```